app/views/pages/terms-and-conditions.html.haml

```diff
@@ -0,0 +1,98 @@
+- content_for :page_title do
+  Terms and Conditions
+
+.grid-row
+
+  .column-two-thirds
+    %h1.heading-xlarge Terms and Conditions
+
+    %h2.heading-large All Users
+
+    %h3.heading-medium Using the website
+    %p Please read these Terms of Use (“General Terms”) carefully before using this Teaching Jobs website (the “Service”).
+    %P This Service  is maintained for your personal use and viewing. Access and use by you of this Service constitutes your acceptance of these General Terms . This takes effect from the date on which you first use this website. If you do not agree to these terms, you must not use this website.
+    %p If we change these General Terms we will post the revised document here with an updated effective date. If we make significant changes to these terms, we may notify you by other means such as sending an email or posting a notice on our home page.
+
+    %h3.heading-medium Information About Us
+    %p This Service is operated by [the Department for Education] (“<strong>we</strong>”, “<strong>our</strong>”, or “<strong>us</strong>”). We are a central government department and have our registered office at Sanctuary Buildings, 20 Great Smith Street, SW1P 3BT.
+    %p You can contact us using the following email address: <a href="mailto: #{t('help.email')}">#{t('help.email')}</a>
+
+    %h3.heading-medium Access to the Site
+    %p We shall not be liable if for any reason the Site is unavailable at any time or for any period
```

Comment:
Missing full stop?

Reply:
Yes seems to have been missed in document https://docs.google.com/document/d/1-wOjmgZxcGJncqv7KnrQkFtXIeBWJb1LeNrezTlnLgA/edit?ts=5b08313c#